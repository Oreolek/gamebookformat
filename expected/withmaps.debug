BEGIN DEBUG OUTPUT
Book title: Example Gamebook With Maps
Number of sections: 99

Turn to 1 to begin.
1 () - (EMPTY)
2 () - (EMPTY)
3 () - (EMPTY)
4 () - (EMPTY)
5 () - (EMPTY)
6 () - (EMPTY)
7 () - (EMPTY)
8 () - (EMPTY)
9 () - (EMPTY)
10 (start) -  This should have number 10, as specified in first map file. The end, at 99, should be at 99. 
11 () - (EMPTY)
12 () - (EMPTY)
13 () - (EMPTY)
14 () - (EMPTY)
15 () - (EMPTY)
16 () - (EMPTY)
17 () - (EMPTY)
18 () - (EMPTY)
19 () - (EMPTY)
20 () - (EMPTY)
21 () - (EMPTY)
22 () - (EMPTY)
23 () - (EMPTY)
24 () - (EMPTY)
25 () - (EMPTY)
26 () - (EMPTY)
27 () - (EMPTY)
28 () - (EMPTY)
29 () - (EMPTY)
30 () - (EMPTY)
31 () - (EMPTY)
32 () - (EMPTY)
33 () - (EMPTY)
34 () - (EMPTY)
35 () - (EMPTY)
36 () - (EMPTY)
37 () - (EMPTY)
38 () - (EMPTY)
39 () - (EMPTY)
40 () - (EMPTY)
41 () - (EMPTY)
42 () - (EMPTY)
43 () - (EMPTY)
44 () - (EMPTY)
45 () - (EMPTY)
46 () - (EMPTY)
47 () - (EMPTY)
48 () - (EMPTY)
49 () - (EMPTY)
50 () - (EMPTY)
51 () - (EMPTY)
52 () - (EMPTY)
53 () - (EMPTY)
54 () - (EMPTY)
55 () - (EMPTY)
56 () - (EMPTY)
57 () - (EMPTY)
58 () - (EMPTY)
59 () - (EMPTY)
60 () - (EMPTY)
61 () - (EMPTY)
62 () - (EMPTY)
63 () - (EMPTY)
64 () - (EMPTY)
65 () - (EMPTY)
66 () - (EMPTY)
67 () - (EMPTY)
68 () - (EMPTY)
69 () - (EMPTY)
70 () - (EMPTY)
71 () - (EMPTY)
72 () - (EMPTY)
73 () - (EMPTY)
74 () - (EMPTY)
75 () - (EMPTY)
76 () - (EMPTY)
77 () - (EMPTY)
78 () - (EMPTY)
79 () - (EMPTY)
80 () - (EMPTY)
81 () - (EMPTY)
82 () - (EMPTY)
83 () - (EMPTY)
84 () - (EMPTY)
85 () - (EMPTY)
86 () - (EMPTY)
87 () - (EMPTY)
88 () - (EMPTY)
89 () - (EMPTY)
90 () - (EMPTY)
91 () - (EMPTY)
92 () - (EMPTY)
93 () - (EMPTY)
94 () - (EMPTY)
95 () - (EMPTY)
96 () - (EMPTY)
97 () - (EMPTY)
98 () - (EMPTY)
99 (end) -  Number 99, as specified in second map file.
END DEBUG OUTPUT
