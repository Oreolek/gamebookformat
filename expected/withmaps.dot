digraph gamebook {

 10->99
}
