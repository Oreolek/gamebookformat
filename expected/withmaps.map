{"Dummy": 98, "end": 99, "start": 10}