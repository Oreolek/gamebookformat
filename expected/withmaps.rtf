{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Example Gamebook With Maps
\b0\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s10}10{\*\bkmkend s10}
\b0\
\ql  This should have number 10, as specified in first map file. The end, at \b {\field{\*\fldinst  REF s99 \\h }{\fldrslt 99}}
\b0
, should be at 99.  \
\

\b \qc {\*\bkmkstart s99}99{\*\bkmkend s99}
\b0\
\ql  Number 99, as specified in second map file. \
\

}
