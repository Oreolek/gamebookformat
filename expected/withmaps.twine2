<tw-storydata name="Example Gamebook With Maps" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="10" name="10" tags="" position="100,100"> This should have number 10, as specified in first map file. The end, at [[99]]
, should be at 99. </tw-passagedata>
<tw-passagedata pid="99" name="99" tags="" position="990,990"> Number 99, as specified in second map file.</tw-passagedata>
</tw-storydata>
