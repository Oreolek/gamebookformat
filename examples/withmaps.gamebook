title = Example Gamebook With Maps
author = Pelle Nilsson

* start
This should have number 10, as specified in first map file.
The end, at [[end]], should be at 99.

* end
Number 99, as specified in second map file.
