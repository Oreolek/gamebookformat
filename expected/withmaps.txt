    Example Gamebook With Maps


Turn to 1 to begin.
10
 This should have number 10, as specified in first map file. The end, at 99, should be at 99. 

99
 Number 99, as specified in second map file.

